base64.js

```diff
@@ -3,19 +3,31 @@
 //
 // THIS FILE IS AUTOMATICALLY GENERATED! DO NOT EDIT BY HAND!
 //
-;(function (global, factory) {
+;(function(global, factory) {
     typeof exports === 'object' && typeof module !== 'undefined'
         ? module.exports = factory(global)
         : typeof define === 'function' && define.amd
-        ? define(factory) : factory(global)
-}(( typeof self !== 'undefined' ? self
+        ? define(factory) : 
+        // cf. https://github.com/dankogai/js-base64/issues/119
+        (function() {
+            global = global || {}; // existing version for noConflict()
+            const _Base64 = global.Base64;
+            const gBase64 = factory()
+            gBase64.noConflict = () => {
+                global.Base64 = _Base64;
+                return gBase64;
+            };
+            if (global.Meteor) { // Meteor.js
```

Comment:
@dankogai Question here, why `Meteor` should be considered specially? Should `global.Base64 = gBase64` same as `Base64 = gBase64` here?